picard/file.py

```diff
@@ -296,6 +299,16 @@ def _save_and_rename(self, old_filename, metadata):
             return None
         new_filename = old_filename
         if not config.setting["dont_write_tags"]:
+            # Copy file to memory
+            try:
+                num_bytes = os.stat(old_filename).st_size
```

Comment:
IMHO this would be a bit easier to read (requires less context) by using `new_filename` here and in the rest of this changeset.

Reply:
Made a few changes and should be more clear. :)